### PR TITLE
Add V0 Ozone Diagnostics

### PR DESCRIPTION
**Pull request by shawnusaf**
Adds V0 ozone diagnostics that output ozone seasonal cycle and profile plots with model comparisons to ozone sonde climatology.

### CODE REVIEW

**Comment by shawnusaf** [comment]
Addresses https://github.com/NCAR/ADF/issues/281

**Review by justin-richling — CHANGES_REQUESTED** [review]
Thanks for the work @shawnusaf, this is great to have. I just have some small changes for you to fix after which should be good to go. 

The biggest change requested is the move to using geocat for the hybrid to pressure interpolation. I have the updated code in the review, but let me know if you have any questions about it.

Let me know when you get the changes in and well move forward, thanks!

**Comment by shawnusaf** [comment]
Changes have been addressed in the newly updated ozone_diagnostics.py file.  I have also added in taylor diagrams to the code.

File: config_cam_baseline_example.yaml [hunk 1]
```diff
@@ -58,7 +58,7 @@
 # Note that the string 'USER-NAME-NOT-SET' is used in the jupyter script
 # to check for a failure to customize
 #
-user: 'USER-NAME-NOT-SET'
+user: 'shawnh'
```
Comment (justin-richling) [inline config_cam_baseline_example.yaml:61]: This just needs to be changed back

File: scripts/plotting/ozone_diagnostics.py [hunk 2]
```diff
@@ -0,0 +1,875 @@
...
+   baseline_climo_loc = adfobj.get_baseline_info('cam_climo_loc',required=True)
+   ncases = len(cam_climo_loc)
+   
+   #check the number of cases.  For the O3 diagnostics there needs to be 2.
+   #If there is more than 2, then only the first two will be used.
+   case_test = adfobj.get_cam_info('cam_case_name', required=True)[0]
+   case_base = adfobj.get_baseline_info('cam_case_name',required=True)
+   
+   #Grab all case nickname(s)
+   test_nicknames = adfobj.case_nicknames["test_nicknames"]
```
[the remark this answers is not in the document]
Comment (justin-richling) [reply scripts/plotting/ozone_diagnostics.py:866]: @shawnusaf I would change this line so the plot legend isn't a list for the test case
Screenshot 2024-10-30 at 1 15 30 PM (3)

**Comment by justin-richling** [comment]
@shawnusaf I left just a couple of suggestions, after that it should be good to merge in.

**Pull request merged**